azurerm_cdn_endpoint/main.tf

```diff
@@ -23,6 +23,34 @@ resource "azurerm_cdn_endpoint" "cdn_endpoint" {
     host_name = var.origin_host_name
   }
 
+  dynamic "global_delivery_rule" {
+    iterator = l
```

Comment:
You can't have more than 1 block of type global_delivery_rule (you can't use a list).

Reply:
@francescopersico Right! Actually, it's possible to have at most one global delivery rule with multiple different actions but in our case I have defined the support to add a specific action type (Cache expiration action)... so, only one is allowed. I removed the list.